gazebo_ros_control:
  pid_gains:
    joint1: {p: 200, i: 0.0, d: 0.1}
    joint2: {p: 200, i: 0.0, d: 0.1}
